applications/nrf_desktop/sample.yaml

```diff
@@ -85,7 +85,6 @@ tests:
       - nrf52840dk/nrf52840
       - nrf52840gmouse/nrf52840
```

Comment:
Can we also drop `nrf54l15dk/nrf54l10/cpuapp` from `applications.nrf_desktop.zdebug_fast_pair.gmouse.uart`'s `integration_platforms`?

Seems like we missed removing `kmu_provision` suffix in the `quarantine_integration.yaml` while changing test names